src/main/java/com/tfgm/models/TramStop.java

```diff
@@ -5,75 +5,33 @@
 import java.util.LinkedList;
```

Comment:
Looks good, but why did you remove the JavaDocs?

Reply:
unsure, added them back in now